server/src/main/java/org/opensearch/rest/BaseRestHandler.java

```diff
@@ -200,6 +202,32 @@ protected final String unrecognized(
         return Collections.emptySet();
     }
 
+    /**
+     * Parse the deprecated request parameter 'master_timeout', and add deprecated log if the parameter is used.
+     * It also validates whether the value of 'master_timeout' is the same with 'cluster_manager_timeout'.
+     * Remove the method along with MASTER_ROLE.
+     * @deprecated As of 2.0, because promoting inclusive language.
+     * @param mnr the action request
+     * @param request the REST request to handle
+     * @param logger the logger that logs deprecation notices
+     * @param logMsgKeyPrefix the key prefix of a deprecation message to avoid duplicate messages.
+     */
+    @Deprecated
```

Comment:
Nitpick, while the parameter is deprecated, I don't think _this method_ itself is deprecated because it is still required to be used during the compatibility period. This can result in warnings or things being flagged in an IDE, suggesting to a developer that it can be removed, but it can't be safely removed until support for the previous names is actually removed. Not a big deal though...

Reply:
Hi @andrross Thanks a lot for your review! I understand it's not accurate to deprecate this method, my idea is telling others this is temporary and not to use this method except me 😂, while I don't know a suitable notice.
I will remove the annotation, removing it's not a big deal.

Reply:
Removed the deprecate annotation.